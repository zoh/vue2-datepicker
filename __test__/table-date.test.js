import { mount } from '@vue/test-utils';
import TableDate from '../src/calendar/table-date.vue';

let wrapper;

afterEach(() => {
  wrapper.destroy();
});

describe('TableDate', () => {
  it('corrent render', () => {
    wrapper = mount(TableDate, {
      propsData: {
        calendarYear: 2019,
        calendarMonth: 9,
        firstDayOfWeek: 1,
        titleFormat: 'DD/MM/YYYY',
      },
    });
    expect(wrapper.element).toMatchSnapshot();
  });
});
