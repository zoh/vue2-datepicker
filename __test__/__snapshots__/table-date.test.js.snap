// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableDate corrent render 1`] = `
<table
  class="mx-table mx-table-date"
  firstdayofweek="1"
>
  <thead>
    <tr>
      <!---->
       
      <th>
        Su
      </th>
      <th>
        Mo
      </th>
      <th>
        Tu
      </th>
      <th>
        We
      </th>
      <th>
        Th
      </th>
      <th>
        Fr
      </th>
      <th>
        Sa
      </th>
    </tr>
  </thead>
   
  <tbody>
    <tr
      class="mx-date-row"
    >
      <!---->
       
      <td
        class="cell"
        data-day="-1"
        title="29/09/2019"
      >
        <div>
          29
        </div>
      </td>
      <td
        class="cell"
        data-day="0"
        title="30/09/2019"
      >
        <div>
          30
        </div>
      </td>
      <td
        class="cell"
        data-day="1"
        title="01/10/2019"
      >
        <div>
          1
        </div>
      </td>
      <td
        class="cell"
        data-day="2"
        title="02/10/2019"
      >
        <div>
          2
        </div>
      </td>
      <td
        class="cell"
        data-day="3"
        title="03/10/2019"
      >
        <div>
          3
        </div>
      </td>
      <td
        class="cell"
        data-day="4"
        title="04/10/2019"
      >
        <div>
          4
        </div>
      </td>
      <td
        class="cell"
        data-day="5"
        title="05/10/2019"
      >
        <div>
          5
        </div>
      </td>
    </tr>
    <tr
      class="mx-date-row"
    >
      <!---->
       
      <td
        class="cell"
        data-day="6"
        title="06/10/2019"
      >
        <div>
          6
        </div>
      </td>
      <td
        class="cell"
        data-day="7"
        title="07/10/2019"
      >
        <div>
          7
        </div>
      </td>
      <td
        class="cell"
        data-day="8"
        title="08/10/2019"
      >
        <div>
          8
        </div>
      </td>
      <td
        class="cell"
        data-day="9"
        title="09/10/2019"
      >
        <div>
          9
        </div>
      </td>
      <td
        class="cell"
        data-day="10"
        title="10/10/2019"
      >
        <div>
          10
        </div>
      </td>
      <td
        class="cell"
        data-day="11"
        title="11/10/2019"
      >
        <div>
          11
        </div>
      </td>
      <td
        class="cell"
        data-day="12"
        title="12/10/2019"
      >
        <div>
          12
        </div>
      </td>
    </tr>
    <tr
      class="mx-date-row"
    >
      <!---->
       
      <td
        class="cell"
        data-day="13"
        title="13/10/2019"
      >
        <div>
          13
        </div>
      </td>
      <td
        class="cell"
        data-day="14"
        title="14/10/2019"
      >
        <div>
          14
        </div>
      </td>
      <td
        class="cell"
        data-day="15"
        title="15/10/2019"
      >
        <div>
          15
        </div>
      </td>
      <td
        class="cell"
        data-day="16"
        title="16/10/2019"
      >
        <div>
          16
        </div>
      </td>
      <td
        class="cell"
        data-day="17"
        title="17/10/2019"
      >
        <div>
          17
        </div>
      </td>
      <td
        class="cell"
        data-day="18"
        title="18/10/2019"
      >
        <div>
          18
        </div>
      </td>
      <td
        class="cell"
        data-day="19"
        title="19/10/2019"
      >
        <div>
          19
        </div>
      </td>
    </tr>
    <tr
      class="mx-date-row"
    >
      <!---->
       
      <td
        class="cell"
        data-day="20"
        title="20/10/2019"
      >
        <div>
          20
        </div>
      </td>
      <td
        class="cell"
        data-day="21"
        title="21/10/2019"
      >
        <div>
          21
        </div>
      </td>
      <td
        class="cell"
        data-day="22"
        title="22/10/2019"
      >
        <div>
          22
        </div>
      </td>
      <td
        class="cell"
        data-day="23"
        title="23/10/2019"
      >
        <div>
          23
        </div>
      </td>
      <td
        class="cell"
        data-day="24"
        title="24/10/2019"
      >
        <div>
          24
        </div>
      </td>
      <td
        class="cell"
        data-day="25"
        title="25/10/2019"
      >
        <div>
          25
        </div>
      </td>
      <td
        class="cell"
        data-day="26"
        title="26/10/2019"
      >
        <div>
          26
        </div>
      </td>
    </tr>
    <tr
      class="mx-date-row"
    >
      <!---->
       
      <td
        class="cell"
        data-day="27"
        title="27/10/2019"
      >
        <div>
          27
        </div>
      </td>
      <td
        class="cell"
        data-day="28"
        title="28/10/2019"
      >
        <div>
          28
        </div>
      </td>
      <td
        class="cell"
        data-day="29"
        title="29/10/2019"
      >
        <div>
          29
        </div>
      </td>
      <td
        class="cell"
        data-day="30"
        title="30/10/2019"
      >
        <div>
          30
        </div>
      </td>
      <td
        class="cell"
        data-day="31"
        title="31/10/2019"
      >
        <div>
          31
        </div>
      </td>
      <td
        class="cell"
        data-day="32"
        title="01/11/2019"
      >
        <div>
          1
        </div>
      </td>
      <td
        class="cell"
        data-day="33"
        title="02/11/2019"
      >
        <div>
          2
        </div>
      </td>
    </tr>
    <tr
      class="mx-date-row"
    >
      <!---->
       
      <td
        class="cell"
        data-day="34"
        title="03/11/2019"
      >
        <div>
          3
        </div>
      </td>
      <td
        class="cell"
        data-day="35"
        title="04/11/2019"
      >
        <div>
          4
        </div>
      </td>
      <td
        class="cell"
        data-day="36"
        title="05/11/2019"
      >
        <div>
          5
        </div>
      </td>
      <td
        class="cell"
        data-day="37"
        title="06/11/2019"
      >
        <div>
          6
        </div>
      </td>
      <td
        class="cell"
        data-day="38"
        title="07/11/2019"
      >
        <div>
          7
        </div>
      </td>
      <td
        class="cell"
        data-day="39"
        title="08/11/2019"
      >
        <div>
          8
        </div>
      </td>
      <td
        class="cell"
        data-day="40"
        title="09/11/2019"
      >
        <div>
          9
        </div>
      </td>
    </tr>
  </tbody>
</table>
`;
